lib/Core/SQLiteBuildDB.cpp

```diff
@@ -193,7 +212,12 @@ class SQLiteBuildDB : public BuildDB {
       db, insertIntoRuleDependenciesStmtSQL,
       -1, &insertIntoRuleDependenciesStmt, nullptr);
     assert(result == SQLITE_OK);
-
+    
+    result = sqlite3_prepare_v2(
+      db, deleteFromKeysStmtSQL,
+      -1, &deleteFromKeysStmt, nullptr);
+    assert(result == SQLITE_OK);
+    
```

Comment:
This is unused in this patch... I am ok with landing this without removing keys, we should file a follow on to track that, but in practice I don't see it being a huge problem in the short term.